tensorflow-examples/src/main/java/org/tensorflow/model/examples/tensors/TensorCreation.java

```diff
@@ -0,0 +1,62 @@
+package org.tensorflow.model.examples.tensors;
+
+import org.tensorflow.Tensor;
+import org.tensorflow.tools.Shape;
+import org.tensorflow.tools.ndarray.IntNdArray;
+import org.tensorflow.tools.ndarray.NdArrays;
+import org.tensorflow.types.TInt32;
+
+/**
+ * Creates a few tensors of ranks: 0, 1, 2, 3.
+ */
+public class TensorCreation {
+    public static void main(String[] args) {
+        // Rank 0 Tensor
+        Tensor<TInt32> rank0Tensor = TInt32.scalarOf(42);
+        System.out.println("---- Scalar tensor ---------");
+        System.out.println("DataType: " + rank0Tensor.dataType().name());
+        System.out.println("Rank: " + rank0Tensor.data().rank());
+        System.out.println("NumElements: " + rank0Tensor.data().size());
+
+        // Rank 1 Tensor
+        Tensor<TInt32> rank1Tensor = TInt32.vectorOf(1, 2, 3, 4, 5, 6, 7, 8, 9, 10);
+        System.out.println("---- Vector tensor ---------");
+        System.out.println("DataType: " + rank1Tensor.dataType().name());
+        System.out.println("Rank: " + rank1Tensor.data().rank());
+        System.out.println("NumElements: " + rank1Tensor.data().size());
+
+        // Rank 2 Tensor
+        // 3x2 matrix of ints.
+        IntNdArray matrix2d = NdArrays.ofInts(Shape.of(3, 2));
+
+        matrix2d.set(NdArrays.vectorOf(1, 2), 0)
+                .set(NdArrays.vectorOf(3, 4), 1)
+                .set(NdArrays.vectorOf(5, 6), 2);
+
+        Tensor<TInt32> rank2Tensor = TInt32.tensorOf(matrix2d);
+
```

Comment:
Nit: I suggest some consistency with the usage of the blank lines in your example, sometimes the `System.out` prints are preceded by one and sometimes they are not (sorry for being that picky :-/...)

Reply:
Agree